site/en/adminGuide/monitor/metrics_dashboard.md

```diff
@@ -0,0 +1,163 @@
+| Search Group Top_K  | The 99th percentile of the `Top_K` of search requests combined and executed by each query node within the past two minutes.  | ```  histogram_quantile(0.99, sum by (le, pod, node_id) (rate(milvus_querynode_search_group_topk_bucket{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}[2m])))  ```  | `milvus_querynode_load_segment_latency_bucket`  | The `Top_K` of search requests combined from different buckets .  |
+| Evicted Read Requests Rate  | The number of read requests evicted per minute by each query node within the past two minutes.  | ```  sum(increase(milvus_querynode_read_evicted_count{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}[2m])/2) by (pod, node_id)  ```  | `milvus_querynode_sq_req_count`  | The accumulated number of read requests evicted by query node due to traffic restriction.  |
+  
+</details>
+
+
+<details><summary><h2>Data coordinator</h2></summary>
+  
+  | Panel  | Panel description  | PromQL (Prometheus query lanage)  | The Milvus metrics used  | Milvus metrics description  |
+|---|---|---|---|---|
+| Data Node Num  | The number of data nodes managed by data coord.  | ```  sum(milvus_datacoord_datanode_num{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}) by (app_kubernetes_io_instance)  ```  | `milvus_datacoord_datanode_num`  | The number of data nodes managed by data coord.  |
+| Segment Num  | The number of all types of segments recorded in metadata by data coord.    | ```  sum(milvus_datacoord_segment_num{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}) by (segment_state)  ```  | `milvus_datacoord_segment_num`  | The number of all types of segments recorded in metadata by data coord.    <br/>    Types of segment include: dropped, flushed, flushing, growing, and sealed.    |
+| Collection Num  | The number of collections recorded in metadata by data coord.    | ```  sum(milvus_datacoord_collection_num{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}) by (app_kubernetes_io_instance)  ```  | `milvus_datacoord_collection_num`  | The number of collections recorded in metadata by data coord.    |
+| Stored Rows  | The accumulated number rows of valid and flushed data in data coord.    | ```  sum(milvus_datacoord_stored_rows_num{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}) by (app_kubernetes_io_instance)  ```  | `milvus_datacoord_stored_rows_num`  | The accumulated number rows of valid and flushed data in data coord.    |
```

Comment:
Suppose it should be "The accumulated number of rows of..."

Reply:
Thanks. Corrected.